Widget e Validator para escolha das cidades do Brasil

ribeiropaulor opened the pull request:
- os dois SELECT (uf e cidade) se comunicam via AJAX com o modulo br_cidade.
- o validator retorna o código da cidade;

ribeiropaulor committed:
- Adicionado widget e validator que adicionam a possibilidade de se ter Estado/Cidade no seu form. O filtro de cidades funciona via Ajax.

ribeiropaulor committed:
- Adicionado widget e validator que adicionam a possibilidade de se ter Estado/Cidade no seu form. O filtro de cidades funciona via Ajax.

ribeiropaulor committed:
- Adicionado widget e validator que adicionam a possibilidade de se ter Estado/Cidade no seu form. O filtro de cidades funciona via Ajax.

rafaelgou commented:
Rapaz, olhei o código "porcimisticamente", muito bom!

Durante a semana tenho vou verificar e validar e agregar seu pull.

Depois divulgo na lista do symfony-pt.

[]'s

Rafael Goulart

ribeiropaulor committed:
Validators sfValidatorDDD , sfValidatorSoftCep e sfValidatorTelefone adicionados.  Adicionam validacao
para DDD de telefone, para telefones no formato (99)9999-9999 e para Cep (modo soft - somente formato numerico de 8 digitos.)

ribeiropaulor committed:
Documentacao atualizada.

ribeiropaulor committed:
Bug fixed: quando utilizando em um FormFilter o parametro $value é uma array(). A validacao estava falhando por conta disso.

ribeiropaulor committed:
Bug fixed: quando utilizando em um FormFilter o parametro $value é uma array(). A validacao estava falhando por conta disso.

ribeiropaulor committed:
Bug fixed: quando utilizando em um FormFilter o parametro $value é uma array(). A validacao estava falhando por conta disso.

ribeiropaulor committed:
Novidade: adicionada uma tarefa para formatar o CPF/CNPJ de uma coluna na tabela utilizando o doctrine como ORM.

ribeiropaulor committed:
Novidade: adicionada uma tarefa para formatar o CPF/CNPJ de uma coluna na tabela utilizando o doctrine como ORM.

ribeiropaulor committed:
Bug fix: quando o campo nao eh required estah tentando validar para a string vazia
